.github/workflows/workflow.yml

```diff
@@ -0,0 +1,19 @@
+name: Validate and tests
+
+on:
+  push
+
+jobs:
```

Comment:
You could also add a step to pip _black_ and run a `black --check` if you want that your module is _black complient_

Reply:
Oh yes.